Source/WebCore/svg/animation/SVGSMILElement.cpp

```diff
@@ -213,6 +213,14 @@ void SVGSMILElement::buildPendingResource()
         svgTarget->addReferencingElement(*this);
 }
 
+bool SVGSMILElement::hasPresentationalHintsForAttribute(const QualifiedName& name) const
+{
+    // Don't map 'fill' to the 'fill' property for animation elements.
```

Comment:
This could use a spec link; I can't tell from the comment if it's for spec or performance or other reasons.

Reply:
Sure - will add 'spec' link (similar to commit message) after EWS run before merging. :-)